SilMock/Google/Service/Directory/UsersResource.php

```diff
@@ -356,12 +356,14 @@ public function listUsers($parameters = [])
             $userEntry = json_decode($userRecord['data'], true);
             if ($this->doesUserMatch($userEntry, $parameters['query'])) {
                 /** @var \Google_Service_Directory_UserName $newName */
+                $nameEntry = $userEntry['name'] ?? null;
+                $givenName = $nameEntry['givenName'] ?? null;
+                $familyName = $nameEntry['familyName'] ?? null;
+                $fullName = $nameEntry['fullName'] ?? trim($givenName . ' ' . $familyName);
```

Comment:
What if givenName or familyName are null?

Reply:
Apparently using the concatenate operator on null doesn't throw an error, but treats it as an empty string. (I had to test this to know.)

Reply:
Thanks, I tried it online and I got an error, but maybe it didn't like how I tried it.

Reply:
string coalesce for the win. ;)